### ISSUE TITLE
Add more collaborators or transfer this repository

### ISSUE THREAD

**Issue by evanslify** [localmed/api-mock#55]
This repository fails on install for now and the original maintainers of this seems not able to accept/resolve issues for now. I recommend that new collaborators shall be added, or this repository's ownership to be transferred.

**Comment by evanslify** [comment]
Ping. @ecordell

**Comment by ecordell** [comment]
Hi @evanslify, sorry for the delay.

So here's a little backstory: I wrote this quickly a couple of years ago with the intention of using it, but never ended up doing that. It's under localmed's namespace, as I was working there at the time. During that time I've tried to merge PRs that claim to fix compatibility with newer protagonist versions, but I wasn't doing what a maintainer should do and testing those claims, so keeping up hasn't been very successful. 

I'm no longer at LocalMed and I don't think they use this either. I'd be happy to transfer ownership personally but I'm not comfortable doing that on a project in localmed's namespace. Maybe @petebrowne could weigh in on whether or not he'd be willing to do that?

That said, there are lots of similar projects listed on [API Blueprint's website](https://apiblueprint.org/tools.html#mock servers) though I admit I've never tried them - is there something about this particular take on the idea that makes it worth maintaining over the other projects?

All this project does is parse the blueprint (using protagonist), and then walk through the AST to add routes to a standard express server. It would be easy to replicate or fork if that's desirable?

**Comment by evanslify** [comment]
Hey @ecordell Thanks for your reply!

I don't have any particular idea, but since this project is listed on npm
and you have to install it to know it's broken due to outdated
dependencies, it would be great if we could do something to improve this
situation. As you mentioned, there are several similar projects listed on
apib's website, are as wonderful as your project (and I use some of them on
daily basis,) maybe we should turn this project into maintenance mode, or
at least edit `README.md` to indicate that this project is not under active
development?

Regardless of your choice, I could help out with a PR.

**Comment by petebrowne** [comment]
Yes, I would happily transfer ownership, or at the very least add some collaborators - we're not using the project anymore

**Comment by petebrowne** [comment]
@evanslify Are you interested in helping out?
